sci-biology/imagej/imagej-9999.ebuild

```diff
@@ -0,0 +1,116 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit git-r3 java-pkg-2 java-ant-2 desktop
+
+"
+# plugins are under a different licenses and can be installed into user's $IJ_HOME/plugins
+#	plugins? ( https://imagej.nih.gov/ij/plugins/ )
+# License: Public-domain
+# Disclaimer: http://imagej.net/disclaimer.html
+"
+
+DESCRIPTION="Image Processing and Analysis in Java"
+
+HOMEPAGE="
+	https://imagej.nih.gov/ij/
+	https://github.com/imagej
+"
+EGIT_REPO_URI="https://github.com/imagej/ImageJ"
+SRC_URI="https://imagej.nih.gov/ij/images/ImageJ.png"
+
+KEYWORD="**"
+
+LICENSE="public-domain"
+
+SLOT="0"
+
+IUSE="doc debug"
+
+RDEPEND="
+	>=virtual/jre-1.7:*
+	dev-java/java-config
+"
+DEPEND="${RDEPEND}
+	>=virtual/jdk-1.7:*
+	dev-java/ant-core
+	app-arch/unzip
```

Comment:
unzip is probably a BDEPEND, maybe ant-core too.

Reply:
This works